content/en/profiler/profiler_troubleshooting.md

```diff
@@ -317,7 +317,7 @@ If you've configured the profiler and don't see profiles in the profile search p
 
    2. Open the `DD-DotNet-Profiler-Native-<Application Name>-<pid>` log file in the `%ProgramData%\Datadog-APM\logs\DotNet` folder.
 
-   3. Look for `libddprof error: Failed to send profile.` entries: This message means that it can't contact the agent.
+   3. Look for `libddprof error: Failed to send profile.` entries: This message means that it can't contact the agent. Ensure the `DD_TRACE_AGENT_URL` is set to the correct Agent URL, please refer to the [Enabling the .NET Profiler-Configuration][5].
```

Comment:
```suggestion
   3. Look for `libddprof error: Failed to send profile.` entries: This message means that it can't contact the agent. Ensure the `DD_TRACE_AGENT_URL` is set to the correct Agent URL. See [Enabling the .NET Profiler-Configuration][5] for more information.
```